.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
     rev: "v0.4.4"
     hooks:
       - id: ruff
-        args: [ --fix]
+        args: [ --fix, --ignore, "UP006,UP007"]
```

Comment:
instead of ignoring these specifically, we should configure the target Python version: https://docs.astral.sh/ruff/settings/#target-version